README.md

```diff
@@ -1,4 +1,4 @@
-<h3 align="center"><img src="https://i.imgur.com/ZQI2EYz.png" alt="logo" height="100px"></h3>
+<h3 align="center"><img src="https://media.discordapp.net/attachments/940740865129865219/980314867561021530/ZQI2EYz.png?width=1440&height=342" alt="logo" height="100px"></h3>
```

Comment:
Maybe don't use discord for the image CDN lol

Reply:
my bad, updated! :>